### PR TITLE
Add Fly to the hosting comparison

### PR DESCRIPTION
**Pull request by CikiMomogi**
https://fly.io

### CODE REVIEW

**Review by KnorpelSenf — APPROVED** [review]

**Review by Loskir — APPROVED** [review]
Looks great!

File: site/docs/hosting/comparison.md [hunk 1]
```diff
@@ -29,20 +29,21 @@ We therefore recommend you to use a different kind of hosting for more complex b
...
+| ---------------------- | ---------- | ------------------------------------------------------------------------ | --------------------------------------------------------------------------------------- | ------- | --------------------------- | --- | ------------------------------ |
+| Deta                   | Free       | No paid plans yet                                                        | No specific limits                                                                      | ✅       | ❓                           | ❓   |                                |
+| Deno Deploy            | Free       | $10/mo subscription for 5M req and 100 GB; $2/1M req, $0.3/GB network    | [100K req/day, 100 GB/mo, 10 ms CPU-time limit](https://deno.com/deploy/pricing)        | ❌       | ✅                           | ❌   |                                |
+| Fly                    | Free       | $1.94/mo subscription for shared-cpu-1x and 256 MB RAM, $0.02/GB network | [3 shared-cpu-1x 256mb VMs, 160GB/mo, 3GB storage](https://fly.io/docs/about/pricing/)  | ✅       | ✅                           | ❓   |                                |
```
Comment (roj1512) [inline site/docs/hosting/comparison.md:36]: Is that shared-cpu-1x a Fly term or a general one meaning a single shared CPU?

Comment (KnorpelSenf) [reply site/docs/hosting/comparison.md:36]: It seems to be a fly term: https://fly.io/blog/new-vms-more-ram-extra-cpu-and-a-dollar-menu/

**Review by roj1512 — APPROVED** [review]
This is badly good.

**Review by habemuscode — APPROVED** [review]

File: site/docs/hosting/comparison.md (hunk 1 above)
Comment (CikiMomogi) [reply site/docs/hosting/comparison.md:36]: Yup. That's a fly term: https://fly.io/docs/about/pricing/

**Pull request merged**